docs/source/man.rst

```diff
@@ -299,35 +295,36 @@ As the network infrastructure changes, it might be necessary to delete some sour
 Scanning
 --------
 
-Use the ``qpc scan start`` command to run connect and inspection scans on the sources. This command scans all of the host names or IP addresses that are defined in the supplied source, and then writes the report information to a comma separated values (CSV) file. Note: Any ssh-agent connection setup for a target host '
-              'will be used as a fallback if it exists.
+Use the ``qpc scan start`` command to run scans on one or more sources. This command scans all of the host names or IP addresses that are defined in the supplied sources, and then writes the report information to a comma separated values (CSV) file. Each instance of a scan is assigned a unique *identifier* so that it can be viewed later.
```

Comment:
Not true:

> , and then writes the report information to a comma separated values (CSV) file. Each instance of a scan is assigned a unique *identifier* so that it can be viewed later.

Reply:
If each instance of the running of a scan does not get an identifier, can you clarify how/when the identifier is assigned? Thanks.

Reply:
Thanks for the extra info about the upcoming report command, I wondered about that. Removed the CSV info for now. Also added a bit to the statement about the identifier:

This command scans all of the host names or IP addresses that are defined in the supplied sources. Each instance of a scan is assigned a unique *identifier* to identify the scan results, so that the results data can be viewed later.

Reply:
👍